modules/developer-cli-odo-advanced-config.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/developer_cli_odo/configuring-the-odo-cli.adoc
+
+:_content-type: REFERENCE
+[id="developer-cli-odo-advanced-config_{context}"]
+
+= Advanced configuration
+
+You can use the following options for debugging and testing `odo` behavior.
+
+.Environment variables controlling odo behavior
```

Comment:
There are few envvars(e.g. ODO_DEBUG_TELEMETRY_FILE, DEVFILE_PROXY, perhaps PODMAN_CMD and DOCKER_CMD too) that are specifically related to testing, I wonder if we need to show them in the downstream doc.